src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -140,6 +135,26 @@ static bool failed_to_reserve_as_requested(char* base, char* requested_address,
   return true;
 }
 
+void ReservedSpace::clear_members() {
+  _base = NULL;
```

Comment:
This method could call `initialize_members()` with the appropriate argument values, couldn't it?

Reply:
Yes. I tried that as well, but then I called it `set_members()` because calling `initialize_members()` from `clear_members()` felt a bit backwards. The reason I didn't go with this approach was that I preferred the name `initialize_members()`. But I'll make the change and call initialize from clear, it's pretty obvious what is going on.